pkg/virt-launcher/virtwrap/libvirtxml/convert.go

```diff
@@ -0,0 +1,336 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017,2018 Red Hat, Inc.
```

Comment:
As per https://github.com/kubevirt/kubevirt/pull/10645 `, should be Copyright The KubeVirt Authors.`